### ISSUE TITLE
Nonstandard foms in contrib.fom

### ISSUE THREAD

**Issue by adler-j** [odlgroup/odl#1423]
The following foms have some issues:

* `standard_deviation_difference`
* `mean_value_difference`
* `range_difference`
* `blurring`
* `false_structures_mask`

Specifically:

* They are never used in ODL (nor any other code to the best of my knowledge)
* They lack documentation
* Naming is wrong (some take the absolute value of the difference, for example) or missleading (what is a false structure?)

My suggestion is that we remove this untill someone who actually wants to use them or cares wants them in, and then we can add them back with a bit of more though put into them.

I know @ozanoktem wants them to stay in there, but then someone has to fix these issues.

**Comment by kohr-h** [comment]
I agree here. In particular, since the documentation is much better after #1280, the functions will look more mature, so expectations may be higher than before, when they still looked a bit crude.

IMO we should restrict ourselves to FOMs that are uncontroversial as to what they mean and how they're computed. The above mentioned ones are either too broadly named ("blurring") or slightly on the trivial side ("mean value difference"). Anybody could implement those on their own. None of them is really established as a quality measure AFAICT.
